app/src/main/java/com/eveningoutpost/dexdrip/cgm/nsfollow/NightscoutFollow.java

```diff
@@ -150,10 +194,13 @@ public static Retrofit getRetrofitInstance() throws IllegalArgumentException {
                     .addInterceptor(new GzipRequestInterceptor())
                     .build();
 
+            final Gson gson = new GsonBuilder()
+                    .registerTypeAdapterFactory(UNRELIABLE_INTEGER_FACTORY)
+                    .create();
```

Comment:
We may need to add `.serializeNulls()`.

Reply:
`serializeNulls` has an effect on serialization only, not during deserialization. Hence, not needed at that code point.